index.html

```diff
@@ -1987,7 +2024,8 @@ <h5><code>writemultipleproperties</code></h5>
         <h4>Actions</h4>
         <section id="http-baseline-profile-protocol-binding-invokeaction">
           <h5><code>invokeaction</code></h5>
-          <p class="rfc2119-assertion"
+	  <span class="rfc2119-assertion"
+	  <p>
```

Comment:
The `id` attribute needs to be inside the `<p>` tag.

Reply:
got two lines switched around.  fixed